### PR TITLE
Pro 4946 hardening caveat flow

### PR DESCRIPTION
**Pull request by Wivenhoe-HMCTS**
Pro 4946 hardening caveat flow
https://tools.hmcts.net/jira/browse/PRO-4946

[ ] Yes
[x ] No
```

### CODE REVIEW

File: test/component/applicant/testAddress.js [hunk 1]
```diff
@@ -21,11 +21,16 @@ describe('applicant-address', () => {
     });
 
     describe('Verify Content, Errors and Redirection', () => {
-        testHelpBlockContent.runTest('ApplicantAddress');
+        testHelpBlockContent.runTest('ApplicantAddress', true);
```
Comment (LucaDelBuonoHMCTS) [inline test/component/applicant/testAddress.js]: What's that `true` for? The `runTest` method only accepts one param...

File: app/routes.js [hunk 2]
```diff
@@ -54,6 +54,56 @@ router.use((req, res, next) => {
...
+router.get('/*', (req, res, next) => {
+    const formdata = req.session.form;
+    if (!includes(config.whitelistedPagesForStartPageRedirect, req.originalUrl) &&
+        !get(formdata, 'applicant')) {
+        res.redirect('/start-page');
+    } else {
+        next();
+    }
+});
+
```
Comment (LucaDelBuonoHMCTS) [inline app/routes.js]: Aren't these 5 routes all the same?

File: test/component/applicant/testAddress.js (hunk 1 above)
Comment (Wivenhoe-HMCTS) [reply test/component/applicant/testAddress.js]: I'll fix

File: app/routes.js (hunk 2 above)
Comment (Wivenhoe-HMCTS) [reply app/routes.js]: Yes merge issue, ill fix

**Pull request merged**